addons/mail_plugin/views/res_partner_iap_views.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_partner_iap_action" model="ir.actions.act_window">
```

Comment:
Side question: is iap_enrich_info field of partner displayed somewhere ? Shouldn't we have a way to display it other than a logged message ? Maybe in debug mode, to check with LNA. Same question with iap_search_domain, even if it is less useful for people.

Reply:
You can see it in the technical menu, but I don't think it's a good idea to show it for the res.partner model, even in debug mode because it's a very big JSON, and those information are not very useful for the user. 

And we already log the information in a pretty HTML message in the chatter

![screenshot_spect](https://user-images.githubusercontent.com/54316226/113302759-ac019500-9300-11eb-9335-38d8ee8c6b21.png)

(LNA agree to not show it in debug mode for the res.partner)

Reply:
Ok, seems legit.